server/routes/auth/providers/google.ts

```diff
@@ -69,10 +65,6 @@ if (GOOGLE_CLIENT_ID && GOOGLE_CLIENT_SECRET) {
             throw GoogleWorkspaceRequiredError();
           }
 
-          if (!isDomainAllowed(domain)) {
-            throw GoogleWorkspaceInvalidError();
-          }
```

Comment:
I think I need to replicate a bit of this functionality still

I removed this to rely on the logic in `TeamCreator` to not create a new team in the domain wasn't allowed. But I think I also need to make changes in `accountProvisioner` to make sure the domain of the user ALSO matches one of the allowed domains

Reply:
Yep, sounds right